pkg/maps/lbmap/lbmap.go

```diff
@@ -631,7 +631,10 @@ func updateMasterService(fe ServiceKey, v ServiceValue, activeBackends, quaranti
 	})
 	v.SetFlags(flag.UInt16())
 	if sessionAffinity {
-		v.SetSessionAffinityTimeoutSec(sessionAffinityTimeoutSec)
+		if err := v.SetSessionAffinityTimeoutSec(sessionAffinityTimeoutSec); err != nil {
+			log.Warn("Failure in updateMasterService due to error from SetSessionAffinityTimeoutSec: %w")
```

Comment:
unused/invalid format specifier in this log: `%w`

Reply:
Thanks, fixed!